frontend/src/templates/Field/YesNo/YesNoField.stories.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from 'react'
+import { FormProvider, useForm } from 'react-hook-form'
+import { Text } from '@chakra-ui/react'
+import { Meta, Story } from '@storybook/react'
+
+import { BasicField } from '~shared/types/field'
+
+import Button from '~components/Button'
+
+import {
+  YesNoField as YesNoFieldComponent,
+  YesNoFieldProps,
+  YesNoFieldSchema,
+} from './YesNoField'
+
+export default {
+  title: 'Templates/Field/YesNoField',
+  component: YesNoFieldComponent,
+  decorators: [],
+  parameters: {
+    docs: {
+      // Required in this story due to rhf conflicting with Storybook somehow.
+      // See https://github.com/storybookjs/storybook/issues/12747.
+      source: {
+        type: 'code',
+      },
+    },
+  },
+} as Meta
+
+const requiredSchema: YesNoFieldSchema = {
+  title: 'Do you like apples',
```

Comment:
![Screenshot 2021-08-25 at 11 34 46 AM](https://user-images.githubusercontent.com/29480346/130722169-70b511a4-34cb-4eda-a403-5712b2235d12.png)

Reply:
The heck @seaerchin still stalking this repo